### DISCUSSION TITLE
Way points from an ID

### DISCUSSION THREAD

**Discussion by ChildishGiant** [mvexel/overpass-api-python-wrapper#164] [Q&A]
I'm trying to get a way from an ID. I can use `api.get('way(WAYID);(._;>;);out;')` but this doesn't return the points in order. Is there a way I can use the  WayQuery function for this?

**Comment by mvexel** [comment]
If you use the overpass API directly, does it return what you expect?

**Reply by mvexel** [reply]
oh and sorry it took me like 4 months to answer :/
